.bazelrc

```diff
@@ -1,16 +0,0 @@
-# Build with --config=local to send build logs to your local server
```

Comment:
(I think I sent this as a single-review comment so ignore if you've already addressed)

Please revert this part of the change (so we keep .bazelrc in place). Thanks!

Reply:
oops accidentally staged... removed